From 5a5a5c1e7073f21c6036c7b5b32ab9dd514c4e12 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 6 Jul 2018 11:54:54 +0200
Subject: [PATCH] configure.ac: fix --enable-utils

Running "configure --enable-utils" _disables_ compilation/installation of
utils (and vice versa) while omitting the --(en|dis)able-utils option auto-
enables utils.
This patch fixes the logic when the option is given on command line but
keeps utils build enabled when the option was omitted.
---
 configure.ac | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index a1f51d0..f96879a 100755
--- a/configure.ac
+++ b/configure.ac
@@ -25,11 +25,13 @@ fi
 AC_SYS_LARGEFILE
 
 #option: utils
+MAYBE_UTILS="utils"
 AC_ARG_ENABLE([utils],
               [AC_HELP_STRING([--enable-utils],
                               [Build util programs])],
-	      [MAYBE_UTILS=""],
-	      [MAYBE_UTILS="utils"])
+	      [if test $enableval = no ; then
+	    	    MAYBE_UTILS=""
+	       fi])
 AC_SUBST(MAYBE_UTILS)
 
 #option: examples
